discoveryapis_generator/lib/src/pubspec.dart

```diff
@@ -27,11 +27,11 @@ class Pubspec {
   String get sdkConstraint => '>=2.10.0 <3.0.0';
 
   static const dependencies = {
-    'http': '\'>=0.11.1 <0.13.0\'',
-    '_discoveryapis_commons': '\'>=0.1.0 <0.3.0\'',
+    'http': '">=0.12.2 <0.14.0"',
+    '_discoveryapis_commons': '^0.2.1',
   };
 
   static const _defaultDevDependencies = {
-    'test': '^1.0.0',
+    'test': '^1.15.7',
```

Comment:
```suggestion
    'test': '^1.15.0',
```

We typically avoid putting the patch version in the constraint unless we specifically need a bug fix. If this package doesn't use `spawnHybridUri` in any tests we can keep this without the patch version.

Reply:
since it's a dev_dep, I just go w/ the latest version compatible w/ the SDK lower bound. 🤷